integration_test.go

```diff
@@ -27,17 +26,14 @@ import (
 	"golang.org/x/sync/singleflight"
 )
 
-func TestMain(m *testing.M) {
-	cmd := exec.Command("podman", "run", "-p", "19085:9000", "quay.io/minio/minio", "server", "/data")
-	stderrPipe, err := cmd.StderrPipe()
-	if err != nil {
-		panic(err)
-	}
-	err = cmd.Start()
+const containerName string = "ctile_integration_test_minio"
+const testLogSaysPastTheEnd string = "oh no! we fell off the end of the log!"
+
+func startContainer() {
```

Comment:
Since we're now calling this from a test case instead of from TestMain, the `panic`s can become error returns, and the test can check those returns and `t.Fatal`.